packages/cozy-sharing/src/SharingBanner/components/publicBanner.styl

```diff
@@ -0,0 +1,15 @@
+@require 'components/avatar'
+@require 'components/circle'
```

Comment:
It's clearly what we avoid to do. We should not require components/* anymore. We should use React component from UI instead. I know the code already exists, but let's fix this.

Reply:
Agreed

Reply:
I tried to use components instead and it's a bit tricky, it's a markdown string compiled to HTML string by an external lib. I'm guessing that's why it was implemented with CSS import instead of component composition. Not sure how to handle this unless by getting rid of the markdown library